uint8 numSucResp
uint32 curWaitTime